### PR TITLE
CorrDiff: update readme

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

Closes 

## Checklist

- [ ] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [ ] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Review by ktangsali — APPROVED** [review]

File: examples/generative/corrdiff/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ the potential of end-to-end, global-to- km-scales machine learning weather predi
 Refer to the [CorrDiff preprint](https://arxiv.org/abs/2309.15214) for more details.
 
 <p align="center">
-<img src="../../../docs/img/corrdiff_demo.gif" />
+<img src="../../../docs/img/corrdiff_cold_front" />
```
Comment (ktangsali) [inline examples/generative/corrdiff/README.md:23]: This is not rendering presently. https://github.com/mnabian/modulus/tree/enh-ext-corrdiff_update_figure/examples/generative/corrdiff

`.png` is missing. Also, if the `.gif` is no longer being used, can we cleanup?

**Review by ktangsali — APPROVED** [review]

File: examples/generative/corrdiff/README.md (hunk 1 above)
Comment (mnabian) [reply examples/generative/corrdiff/README.md]: Fixed it, thanks!

**Review by yairchn — APPROVED** [review]
LGTM

**Pull request merged**